sklearn/linear_model/logistic.py

```diff
@@ -926,28 +950,28 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
     """Logistic Regression (aka logit, MaxEnt) classifier.
 
     In the multiclass case, the training algorithm uses the one-vs-rest (OvR)
-    scheme if the 'multi_class' option is set to 'ovr' and uses the
-    cross-entropy loss, if the 'multi_class' option is set to 'multinomial'.
-    (Currently the 'multinomial' option is supported only by the 'lbfgs' and
-    'newton-cg' solvers.)
+    scheme if the 'multi_class' option is set to 'ovr' and uses the cross-
+    entropy loss, if the 'multi_class' option is set to 'multinomial'.
+    (Currently the 'multinomial' option is supported only by the 'lbfgs',
+    'sag' and 'newton-cg' solvers.)
 
     This class implements regularized logistic regression using the
-    `liblinear` library, newton-cg and lbfgs solvers. It can handle both
-    dense and sparse input. Use C-ordered arrays or CSR matrices containing
-    64-bit floats for optimal performance; any other input format will be
-    converted (and copied).
+    'liblinear' library, 'newton-cg', 'sag' and 'lbfgs' solvers. It can handle
+    both dense and sparse input. Use C-ordered arrays or CSR matrices
+    containing 64-bit floats for optimal performance; any other input format
+    will be converted (and copied).
 
-    The newton-cg and lbfgs solvers support only L2 regularization with primal
-    formulation. The liblinear solver supports both L1 and L2 regularization,
-    with a dual formulation only for the L2 penalty.
+    The 'newton-cg', 'sag', and 'lbfgs' solvers support only L2 regularization
+    with primal formulation. The 'liblinear' solver supports both L1 and L2
+    regularization, with a dual formulation only for the L2 penalty.
 
     Read more in the :ref:`User Guide <logistic_regression>`.
 
     Parameters
     ----------
     penalty : str, 'l1' or 'l2'
-        Used to specify the norm used in the penalization. The newton-cg and
-        lbfgs solvers support only l2 penalties.
+        Used to specify the norm used in the penalization. The newton-cg, sag
```

Comment:
In the `random_state` parameter, should you mention that it is useful only for the `sag` and `liblinear` solvers?